lib/web/desktop/playback.go

```diff
@@ -219,6 +230,10 @@ func (pp *Player) streamSessionEvents(ctx context.Context, cancel context.Cancel
 				}
 			}
 			return
+		case playSpeed := <-pp.playSpeed:
+			prevDelayFactor := delayFactor
+			delayFactor = 1.0 / playSpeed
+			lastDelay = int64(float32(lastDelay) * (delayFactor / prevDelayFactor))
```

Comment:
Let's keep `lastDelay` equivalent to the `Delay` field of the _last_ event we played.

Why can't we just divide by the playback speed where we do the sleep and leave `lastDelay` alone?

Reply:
This did grow more complicated than it needed to be as I cycled though a couple approaches. I just simplified it quite a bit.